config/application.yml.default.ci

```diff
@@ -0,0 +1,7 @@
+production:
```

Comment:
One idea just to make these suuuuper clear
```suggestion
# These secrets are used in CI to run a production build of assets, they are not used in live production requests
production:
```
my other idea is replace them all with `aaaaa` strings of the appropriate lengths